sklearn/feature_selection/from_model.py

```diff
@@ -169,6 +169,12 @@ def fit(self, X, y=None, **fit_params):
         self.estimator_.fit(X, y, **fit_params)
         return self
 
+    @property
+    def threshold_(self):
+        scores = _get_feature_importances(self.estimator_, self.norm_order)
```

Comment:
When is this expensive? I guess if you have a many classes and many features the reduction to the vector is expensive?

Before, when they used ``sfm.threshold_`` it might have been wrong ;)
We could store the scores in fit and / or transform. If people re-assign ``estimator`` that would go stale, though. So we could support re-assigning ``threshold`` but not ``estimator``?